canonical-web-and-design/ubuntu.com
# [Certified improvements] - update cards on /certified

## Done

- Update cards on `/certified` following [design](https://app.zeplin.io/project/60c1ca887b99f18a380e57e9/screen/60f169f609a79e0cf2039ba5)

## QA

- View the page at: https://ubuntu-com-10063.demos.haus/certified
- Check the cards match the [design](https://app.zeplin.io/project/60c1ca887b99f18a380e57e9/screen/60f169f609a79e0cf2039ba5) - this PR is to update the design only, it has the updated copy from the [copy doc](https://docs.google.com/document/d/1pSkPo5fvgrLb-Tjiq_he9MAvPLFX79xiz-H9s2mSXgI/edit) (the copy in the design is out of date).


## Issue / Card

Fixes [#4256](https://github.com/canonical-web-and-design/web-squad/issues/4256)

## Screenshots

<img width="530" alt="Screenshot 2021-07-22 at 11 30 20" src="https://user-images.githubusercontent.com/58959073/126625822-b388eebd-b91f-4f5d-b52d-aa796b03bbfd.png">
